set-web/src/main/java/eu/europeana/set/web/service/controller/jsonld/SearchUserSetRest.java

```diff
@@ -71,10 +71,14 @@ public ResponseEntity<String> searchUserSet(
       HttpServletRequest request) throws HttpException {
 
     try {
-      
       // authorization
       Authentication authentication = verifyReadAccess(request);
 
+      //TODO: temporary fix to remove later
+      if(ProfileConstants.VALUE_PARAM_STANDARD.equals(profileStr)) {
```

Comment:
we can have multiple profiles in the request string, we need to use replace profileStr.replace(VALUE_PARAM_STANDARD, rofileConstants.VALUE_PARAM_ITEMS_META)